app/parse.py

```diff
@@ -8,8 +15,55 @@ class Quote:
     tags: list[str]
 
 
+QUOTE_FIELDS = [field.name for field in fields(Quote)]
+
+
+def parse_one_quote(quote_soup: BeautifulSoup) -> Quote:
+    text = quote_soup.select_one(".text").text
+    author = quote_soup.select_one(".author").text
+    tags = quote_soup.select_one(".keywords")
+
+    if tags and tags.has_attr("content"):
+        tags_content = tags["content"]
+        if tags_content:
+            tags = tags_content.split(",")
+        else:
+            tags = []
+    else:
+        tags = []
+
+    return Quote(text=text, author=author, tags=tags)
+
+
+def get_all_quotes() -> [Quote]:
+    page = 1
+    res = []
+    while True:
+        web = requests.get(f"{BASE_URL}/page/{page}").content
```

Comment:
There is no error handling for network requests in get_all_quotes. If the request fails (e.g., due to a network error or if the server returns a 404 status code), the program will crash. Adding try-except blocks around the requests.get call can prevent this.